__init__.py

```diff
@@ -10,7 +10,8 @@
 config_parser.read("settings.ini", encoding="utf8")
 
 app = Flask(__name__)
-app.config["AIRTABLE_TABLE_NAME"] = config_parser.get("DEFAULT", "airtable_table_name")
+app.config["THEME_COLOR"] = config_parser.get("THEME-COLOR", "theme_color_gray")
```

Comment:
theme-colorが2色あって、片方しかセットしないなら。

```suggestion
app.config["THEME_COLOR"] = config_parser.get("THEME-COLOR-GRAY", "theme_color_gray")
```